src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -45,12 +44,13 @@ def to_vespa_query(self, marqo_query: MarqoQuery) -> Dict[str, Any]:
                                          'You can create a structured index '
                                          'by `mq.create_index("your_index_name", type="structured")`')
 
-        if isinstance(marqo_query, MarqoTensorQuery):
+        if isinstance(marqo_query, MarqoHybridQuery):       # TODO: Rethink structure so order of checking doesn't matter
```

Comment:
@farshidz Because of the class hierarchy. MarqoHybridQuery inherits both TensorQuery and LexicalQuery. This is something we discussed when we first implement this class. We should revisit this design at some point to see if it makes sense to have MarqoHybridQuery inherits directly from MarqoQuery.

Reply:
You can do `type(marqo_query) is ...` for a strict type check